src/components/StatsBoxGrid/index.tsx

```diff
@@ -1,14 +1,16 @@
-import React from "react"
 import { SimpleGrid } from "@chakra-ui/react"
 
+import type { AllMetricData, MetricReturnData } from "@/lib/types"
+
 import { GridItem } from "./GridItem"
 import { useStatsBoxGrid } from "./useStatsBoxGrid"
 
-export interface IProps {}
-
-const StatsBoxGrid: React.FC<IProps> = () => {
-  const { metrics } = useStatsBoxGrid()
+type StatsBoxGridProps = {
+  data: AllMetricData
```

Comment:
Data loaded in _page_ component now during `getStaticProps`... passed into this component as a prop